services/submission.js

```diff
@@ -1117,6 +1132,41 @@ const isSubmissionPermitted = (aSubmission, userInfo) => {
     throw new Error(ERROR.INVALID_STATS_SUBMISSION_PERMISSION);
 }
 
+class ValidationRecord {
+    // submissionID: string
+    // type: array
+    // scope: array
+    // started: Date
+    // status: string
+    constructor(submissionID, type, scope, status) {
+        this._id = v4();
+        this.submissionID = submissionID;
+        this.type = type;
+        this.scope = scope;
+        this.started = getCurrentTime();
+        this.status = status;
+    }
+    static createValidation(submissionID, validationType, validationScope, status) {
+        return new ValidationRecord(submissionID, validationType, validationScope, status);
+    }
+}
```

Comment:
Can you add a "Validation" type to the graphql schema? This isn't used but would help document the structure of the validation records and would be useful if we want to implement queries for this in the future

Reply:
updated